ts/src/gate.ts

```diff
@@ -6697,7 +6706,7 @@ export default class gate extends Exchange {
             }
             const bodyPayload = (body === undefined) ? '' : body;
             const bodySignature = this.hash (this.encode (bodyPayload), sha512);
-            const timestamp = this.seconds ();
+            const timestamp = this.nonce ();
```

Comment:
@AresArtemius Did you test this with private endpoints? We're changing the scale here from ms to seconds.

Reply:
I'll try to register on these exchanges and get API to test. If it doesn't work somewhere, I'll write.

Regarding seconds, I'll fix it.